### ISSUE TITLE
[speech-to-text] Missing types for `recognizeUsingWebSocket` (formerly `createRecognizeStream`)

### ISSUE THREAD

**Issue by Waldo000000** [watson-developer-cloud/node-sdk#764]
When using the speech to text SDK in TypeScript, it would be helpful if the params object required by `recognizeUsingWebSocket` was typed more specifically than `any` (https://github.com/watson-developer-cloud/node-sdk/blob/master/speech-to-text/v1.ts#L453).

**Comment by dpopp07** [comment]
@Waldo000000 I will implement this change, but it's not my highest priority so it may be some time before I get to it. If you need this done soon, feel free to open a PR and I will take a look at it. Thanks

**Comment by anweshan** [comment]
@dpopp07 if I'm not mistaken the accepted parameters should generally be the same from former `createRecognizeStream` right? I can't find the exact differences but it might help to just have that in comment form the previous accepted parameters to `createRecognizeStream` as accepted in `recognizeUsingWebsocket`

**Comment by germanattanasio** [comment]
What's the status of this issue @dpopp07?

**Comment by dpopp07** [comment]
@germanattanasio This never made it into my backlog but it would be easy - it just involves writing a type definition. I can add this to the work for the next major release.

**Comment by MasterOdin** [comment]
@dpopp07 So looking over `lib/recognize-stream.ts`, should all of these have a concrete definition (defined under top-level `interface Options`), or are they supposed to be optional?
```
  /* Query Params*/
  accessToken: string;
  watsonToken: string;
  model: string;
  languageCustomizationId: string;
  acousticCustomizationId: string;
  baseModelVersion: string;
  xWatsonLearningOptOut: boolean;
  xWatsonMetadata: string;

  /* Opening Message Params */
  contentType: string;
  customizationWeight: number;
  inactivityTimeout: number;
  interimResults: boolean;
  keywords: string[];
  keywordsThreshold: number;
  maxAlternatives: number;
  wordAlternativesThreshold: number;
  wordConfidence: boolean;
  timestamps: boolean;
  profanityFilter: boolean;
  smartFormatting: boolean;
  speakerLabels: boolean;
  grammarName: string;
  redaction: boolean;
  processingMetrics: boolean;
  processingMetricsInterval: number;
  audioMetrics: boolean;
```

And are the last three parameters new and will be added to the API documentation (https://cloud.ibm.com/apidocs/speech-to-text?code=node#recognize-audio-websockets-) when v5 of the SDK is released?

**Comment by dpopp07** [comment]
@MasterOdin Ha, yeah that's a good catch. They should all be optional.

As for those last parameters, they are new-ish but not super new. They are documented in the curl tab of the documentation, not sure why they are not in the Node tab. I can investigate that

**Comment by dpopp07** [comment]
Types have been added and released in version 5.0.0

Thanks to @MasterOdin for his contributions there
